### ISSUE TITLE
example not working with craft cms 3

### ISSUE THREAD

**Issue by crazyx13th** [craftcms/guest-entries#23]
Hi there,

getting crazy! :-)

I have a field "body", but there is nothing stored! Title is working

field
---------------
![grafik](https://user-images.githubusercontent.com/8912561/28840321-ed468e5a-76f6-11e7-98e6-2d554b1e48fa.png)

section
---------------
![grafik](https://user-images.githubusercontent.com/8912561/28840399-21499332-76f7-11e7-80ce-48eb48a0ad75.png)

code snippet
---------------
```
<label for="body">Body</label>
<textarea id="body" name="fields[body]">
    {%- if entry is defined %}{{ entry.body }}{% endif -%}
</textarea>
```

what I'm doing wrong!?

thanks!

**Comment by brandonkelly** [comment]
This ended up being a bug in Guest Entries. Just released v2.0.1 which fixes it!

**Comment by crazyx13th** [comment]
thx!!!!
